examples/react/src/App.js

```diff
@@ -391,6 +405,26 @@ renderLoginButtons() {
           logoStyle={{marginLeft:0}}
           onClick={()=>this.handleLogin("lynx")}
       />
+      <LoginButton provider='portis'
+          buttonStyle={{width:250, marginTop:'24px'}}
+          logoStyle={{marginLeft:0}}
+          onClick={()=>this.handleLogin("portis")}
```

Comment:
I get "Cannot initiate a session, invalid wallet provider or wallet provider ID was passed" for portis. Do you get this too?

Reply:
You'll have to add a `DappId` when initializing the provider, you can get one from Portis.io. The portis modal pops up for me but after submitting my email/password it errors out.